lib/ical/time.js

```
@@ -743,15 +740,13 @@
     }
 
     if ("defineProperty" in Object) {
```

Actually this should fail horribly when not available. IMO we should focus on ES5 targets.